### PR TITLE
WE-490 Replace Carter

### LINKED ISSUE #1107: Upgrade Carter to v6
**Is your feature request related to a problem? Please describe.**
Carter v6 has been released

**Describe the solution you'd like**
The version 6 has some breaking changes at the interfaces. Therefore, some code changes are needed.

### PR DESCRIPTION
**Pull request by steinsiv**
## Fixes
This pull request fixes #1107 

## Description
Carter v5 is outdated and needs to be replaced with minimal WebApi

See Issue: https://github.com/equinor/witsml-explorer/issues/1107 
and closed PR: https://github.com/equinor/witsml-explorer/pull/1106

## Type of change
* Enhancement of existing functionality

## Impacted Areas in Application
* API

## Checklist:

*Communication*
* [x] PR is related to an issue
* [ ] I have made corresponding changes to the documentation

*Code quality*
* [x] Code follows the style guidelines
* [x] I have self-reviewed my code
* [x] No new warnings are generated

*Test coverage*
* [x] Existing tests pass
* [ ] New code is covered by passing tests

### CODE REVIEW

File: Src/WitsmlExplorer.Api/HttpHandlers/LogHandler.cs [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    }
+
+    public static async Task<IResult> GetLargeLogData(
+        string wellUid,
+        string wellboreUid,
+        string logUid,
+        [FromQuery(Name = "startIndex")] string startIndex,
+        [FromQuery(Name = "endIndex")] string endIndex,
+        [FromQuery(Name = "startIndexIsInclusive")] bool startIndexIsInclusive,
+        HttpRequest httpRequest,
```
Comment (lindsve) [inline Src/WitsmlExplorer.Api/HttpHandlers/LogHandler.cs]: Maybe this can be replaced with
```
[FromBody] IEnumerable<string> mnemonics,
```
?

Comment (steinsiv) [reply Src/WitsmlExplorer.Api/HttpHandlers/LogHandler.cs]: Yes that simplifies things, will change thanks!

**Review by janburak — DISMISSED** [review]
- [X] Read issue and PR
- [X] Pulled branch and manually tested
- [X] Verify that PR resolves issue
- [X] Reviewed the code

**Review by lindsve — COMMENTED** [review]
@janburak mentioned a good issue here related to OpenAPI.

Carter uses a setup of metadata classes to configure the openapi specification, but I don't seem to find any of that setup here so I'm not sure if this has been used by WE.

ASP.NET Minimal API on the other hand uses Swashbuckle and annotation of the api routes for this configuration:

Example:
```
app.MapGet("/api/something", SomethingHandler.ReturnSomething)
            .Produces<IEnumerable<Something>>();
```

**Review by janburak — APPROVED** [review]
- [X] Read issue and PR
- [X] Pulled branch and manually tested
- [X] Verify that PR resolves issue
- [X] Reviewed the code

**Pull request merged**